src/assets/scss/_settings.scss

```diff
@@ -204,54 +209,57 @@ $block-grid-max: 8;
 // 4. Base Typography
 // ------------------
 
-$header-font-family: $body-font-family;
-$header-font-weight: $global-weight-normal;
-$header-font-style: normal;
-$header-color: inherit;
-$header-lineheight: 1.4;
-$header-margin-bottom: 0.5rem;
-$header-styles: (
-	small: (
-		'h1': ('font-size': rem-calc(28), 'line-height': math.div(rem-calc(36), rem-calc(28))),
-		'h2': ('font-size': rem-calc(22), 'line-height': math.div(rem-calc(30), rem-calc(22))),
-		'h3': ('font-size': rem-calc(18)),
-		'h4': ('font-size': rem-calc(16)),
-	),
-	medium: (
-		'h1': ('font-size': rem-calc(30), 'line-height': math.div(rem-calc(38), rem-calc(30))),
-		'h2': ('font-size': rem-calc(28), 'line-height': math.div(rem-calc(36), rem-calc(28))),
-		'h3': ('font-size': rem-calc(18)),
-		'h4': ('font-size': rem-calc(16)),
-	),
-	large: (
-		'h1': ('font-size': rem-calc(36), 'line-height': math.div(rem-calc(44), rem-calc(36))),
-		'h2': ('font-size': rem-calc(28), 'line-height': math.div(rem-calc(36), rem-calc(28))),
-		'h3': ('font-size': rem-calc(18)),
-		'h4': ('font-size': rem-calc(16)),
-	),
-);
-$header-text-rendering: optimizelegibility;
+// $header-font-family: $body-font-family;
```

Comment:
Do we want to keep this commented-out code?

Reply:
I'm not sure about that. The original settings file is provided by Foundation, and they expect you to copy it into your source code and modify it from there, which means that when Foundation is updated we have to go through their updated settings file and copy over any updates we might need. It would be nice to remove all the lines we aren't using entirely, but it's also nice to have the lines around still so that it's easier to spot the updates when diffing between our settings and the updated settings file in Foundation.

Ideally we would be removing Foundation and Sass entirely, but the different breakpoints between our Foundation settings and our Tailwind primitives means that converting isn't simple.